package.json

```diff
@@ -39,6 +39,7 @@
     "postinstall": "node ./node_modules/vscode/bin/install"
   },
   "dependencies": {
+    "chai-as-promised": "^7.1.1",
```

Comment:
should this be a dev dependency only?

Reply:
This is automatically added when I npm install the package. What do you think should be changed here?

Reply:
I would just move this dependency in the devdependecies list and fix it to 7.1.1 instead of compatible versions